### DISCUSSION TITLE
Banner Plugin Revival

### DISCUSSION THREAD

**Discussion by ProStashio** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#18] [Ask a Technomancer] [answered]
@Obsidian-TTRPG-Community/technomancer 

I have a 500gp diamond, but I have been asked to direct my question here for the Technomancers to raise the banner plugin from the dead. Was curious if anyone could make something like it?

https://github.com/noatpad/obsidian-banners

**Comment by djbrock** [comment, marked as answer]
Looks like several people have forked this code, and some of them have been worked on recently:

https://github.com/jpbruckler/note-banners
https://github.com/zjgzimmerman/obsidian-banners
https://github.com/DudeThatsErin/obsidian-banners

It might be worth a go doing a manual install with the first of those and seeing if it works for you.

When doing so, don't follow the link in the instructions to the release code, use this link https://github.com/jpbruckler/note-banners/releases/tag/1.3.4

**Reply by ProStashio** [reply]
Interesting, sorry didn't know that's how that worked 😅

Will look into that

**Reply by djbrock** [reply]
> Interesting, sorry didn't know that's how that worked 😅

No worries - there's always new stuff to find out. It's not a guarantee that the forks are in working order or work with the latest version, but the fact that it's been worked on recently gives some confidence.

**Reply by sigrunixia** [reply]
I believe, @ProStashio, that @djbrock is entitled to some portion of that diamond. ;)
